### ISSUE TITLE
Tpetra?: <float>,<complex> multiplication operator not defined

### ISSUE THREAD

**Issue by markyoder** [trilinos/Trilinos#7230]
## Bug Report
@trilinos/Tpetra>

Getting a, "operator* not defined":
`Trilinos/packages/tpetra/core/test/ImportExport2/ImportExport2_CrsSortingUtils.cpp:289:62: error: no match for ‘operator*’ (operand types are ‘double’ and ‘__gnu_cxx::__alloc_traits<std::allocator<std::complex<float> >, std::complex<float> >::value_type’ {aka ‘std::complex<float>’})`

part of:

```
// Unit Test the functionality in Tpetra_Import_Util
TEUCHOS_UNIT_TEST_TEMPLATE_3_DECL( Import_Util, SortCrsEntries, Scalar, LO, GO)
{
```

It's a bit surprising because the error seems to imply that `std:<complex>` does not support simple scalar multiplication. Addition appears to be supported (if I replace `2*x` with `x+x` it seems to work. also, `*=` is supported, so replacing:
```
for (size_type i=0; i<new_num_entries; i++) vals2[i] = 2.*vals[i];
```

```
with the. block,

      for (size_type i=0; i<new_num_entries; i++) {
          vals2[i] = vals[i];
          vals2[i]*=2.0;
            };
```
also fixes the problem.

### Description
What went wrong?  What should have happened?  Do you have an idea what might
fix things?  How has this bug affected you?  What are you trying to accomplish?
What specifically must we demonstrate to call this work complete?

### Steps to Reproduce
1. SHA1:  [insert here]
1. Configure script:  [attach here]
1. Configure log:  [attach here]
1. Build log:  [attach here]
1. Input deck:  [attach here]
1. Do this.
1. Do that.
1. Shake fist angrily at computer.
1. Run log:  [attach here]

**Comment by kddevin** [comment]
Thanks @markyoder.  Can you please post your cmake configuration command?  Thank you.

**Comment by kddevin** [comment]
Actually, this issue was fixed with #6336 in Nov 2019.  
I will assume the report was for an older version of Trilinos.  
Please reopen this issue if it is still a problem.
